app/services/sufia/workflow/workflow_by_admin_set_strategy.rb

```diff
@@ -7,7 +7,7 @@ def initialize(_work, attributes)
 
       # @return [String] The name of the workflow by admin_set to use
       def workflow_name
-        return 'default' unless @admin_set_id
+        return 'default' unless @admin_set_id && Flipflop.enable_mediated_deposit?
```

Comment:
Do we need this feature flipper any more?  Can we say that the way to disable mediated deposit is to set the admin set to use the default (0-step) workflow?